portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -46,7 +46,17 @@
     private final Gson gson = new Gson();
     private final UserService credentials = UserServiceFactory.getUserService();
     private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    private final LanguageServiceClient languageService = LanguageServiceClient.create();
+    private LanguageServiceClient languageService;
+
+    @Override
+    public void init() {
+        try {
+            languageService = LanguageServiceClient.create();
+        } catch (IOException e) {
```

Comment:
If you just print the error without doing anything, you'll get more errors later when you try to use languageService. Better to throw an exception here so the ServletContainer knows this servlet has a problem, and it should mabye abort starting up.

init is not allowed to through IOException, but it can through ServletException, so you can wrap the IOException in a servlet exception like this:

`throw new ServletException("Unable to create LanguageServiceClient", e);`

Reply:
Sounds good, I wasn't sure what to do with the exception since I haven't done much with try/catch/throw so I wasn't sure what to throw or if I should've. I'll change it now, thanks